subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1245,9 +1254,12 @@ uint8_t ll_adv_enable(uint8_t enable)
 					aux->ull.ticks_slot +
 					ticks_slot_overhead_aux;
 				uint32_t ticks_anchor_sync =
-					ticks_anchor_aux +
-					ticks_slot_aux +
-					HAL_TICKER_US_TO_TICKS(EVENT_MAFS_US);
+					ticks_anchor_aux + ticks_slot_aux +
+					HAL_TICKER_US_TO_TICKS(
+						MAX(EVENT_MAFS_US,
+						    EVENT_OVERHEAD_START_US) -
+						EVENT_OVERHEAD_START_US +
+						(EVENT_TICKER_RES_MARGIN_US << 1));
```

Comment:
Please put a comment why do you add `(EVENT_TICKER_RES_MARGIN_US << 1));` to the anchor point for sync PDU.

Reply:
done.